pyproject.toml

```diff
@@ -80,7 +80,7 @@ skip_glob = 'signac/_vendor/*'
 
 [tool.pydocstyle]
 convention = "numpy"
-match = "^((?!\\.sync-zenodo-metadata|setup).)*\\.py$"
+match = "^((?!setup).)*\\.py$"
```

Comment:
Not technically in scope, but since we're modifying this line anyway I think we can get rid of it entirely now. We don't have a setup.py file anymore.